plugins/Scripts/Plugins/Registration/Correct_3D_drift.py

```diff
@@ -22,16 +22,42 @@
 from ij.gui import GenericDialog, YesNoCancelDialog, Roi
 from mpicbg.imglib.image import ImagePlusAdapter
 from mpicbg.imglib.algorithm.fft import PhaseCorrelation
-from org.scijava.vecmath import Point3i
-from org.scijava.vecmath import Point3f
+from javax.vecmath import Point3i
+from javax.vecmath import Point3f
```

Comment:
no, this is historical because i developed the code using java 6.
shall i change it and stage a new pull request or is there some other way to change it?